### ISSUE TITLE
Element.waitFor().stale doesn't work

### ISSUE THREAD

**Issue by BlackEnf0rcer** [WileyLabs/teasy#184]
Since there is a method isStale(), that works fine, I suggest to replace
```
        try {
            this.el.isEnabled();
            return false;
        } catch (StaleElementReferenceException var3) {
            return true;
        }
```
by return isStale().

**Comment by BlackEnf0rcer** [comment]
Problem is because of isEnabled() contains againLocate()
```
    @Override
    public boolean isEnabled() {
        try {
            return wrappedElement.isEnabled();
        } catch (StaleElementReferenceException e) {
            againLocate();
            return isEnabled();
        }
    }
```

**Comment by vefimofff** [comment]
fixed 2.0.2
